app/components/certificate/certificate-display.tsx

```diff
@@ -34,66 +34,71 @@ export default function CertificateDisplay({ title, description, value }: Certif
-            {title}
-          </Heading>
-          <Tooltip label={`Copy ${title}`}>
-            <IconButton
-              backgroundColor="transparent"
-              color="black"
-              size="xs"
-              _hover={{
-                background: 'whitesmoke',
-                color: 'teal.500',
-              }}
-              aria-label="Copy to Clipboard"
-              icon={<CopyIcon fontSize="md" />}
-              onClick={() => onCopy()}
-            />
-          </Tooltip>
-          <Tooltip label={`Download ${title}`}>
-            <IconButton
-              backgroundColor="transparent"
-              color="black"
-              size="xs"
-              _hover={{
-                background: 'brand.500',
-                color: 'white',
-              }}
-              aria-label="Download"
-              icon={
-                <DownloadIcon
-                  fontSize="md"
-                  onClick={() =>
-                    toast({
-                      title: `Downloading ${title}`,
-                      position: 'bottom-right',
-                      status: 'success',
-                    })
-                  }
-                />
-              }
-            />
-          </Tooltip>
-        </HStack>
-        <Text>{description}</Text>
-        <Accordion allowMultiple>
-          <AccordionItem>
-            <AccordionButton>
-              <Box as="span" flex="1" textAlign="left">
-                Show/Hide
-              </Box>
-              <AccordionIcon />
-            </AccordionButton>
-            <AccordionPanel>
-              <Text>{value}</Text>
-            </AccordionPanel>
-          </AccordionItem>
-        </Accordion>
-      </Flex>
+    <Flex
+      flexDirection="column"
+      gap="4"
+      textAlign={{ base: 'center', md: 'left' }}
+      fontSize={{ base: 'sm', md: 'md' }}
+    >
+      <HStack gap="4" alignSelf={{ base: 'center', md: 'flex-start' }}>
+        <Heading as="h4" size="sm">
+          {title}
+        </Heading>
+        <Tooltip label={`Copy ${title}`}>
+          <IconButton
+            backgroundColor="transparent"
+            color="black"
+            size="xs"
+            _hover={{
+              background: 'whitesmoke',
+              color: 'teal.500',
+            }}
+            aria-label="Copy to Clipboard"
+            icon={<CopyIcon fontSize="md" />}
+            onClick={() => onCopy()}
+          />
+        </Tooltip>
+        <Tooltip label={`Download ${title}`}>
+          <IconButton
+            backgroundColor="transparent"
+            color="black"
+            size="xs"
+            _hover={{
+              background: 'brand.500',
+              color: 'white',
+            }}
+            aria-label="Download"
+            icon={
+              <DownloadIcon
+                fontSize="md"
+                onClick={() =>
+                  toast({
+                    title: `Downloading ${title}`,
```

Comment:
Download will be less than a second. Perhaps `Download` can avoid potential misunderstanding.

Reply:
sure that make sense